SentryTestUtils/TestSentryUIDeviceWrapper.swift

```diff
@@ -0,0 +1,26 @@
+import Sentry
+
+public class TestSentryUIDeviceWrapper: SentryUIDeviceWrapper {
```

Comment:
This wasn't rewritten, it was only renamed/moved, but didn't diff correctly. No changes in this file.